opentracing-spring-web-itest/common/src/test/java/io/opentracing/contrib/spring/web/interceptor/itest/common/AbstractBaseITests.java

```diff
@@ -170,6 +170,45 @@ public void testControllerException() throws Exception {
                 span.logEntries().get(0).fields().get("handler.class_simple_name"));
     }
 
+    @Test
+    public void testControllerAsyncException() throws Exception {
+        {
+            getRestTemplate().getForEntity("/asyncException", String.class);
+            Awaitility.await().until(reportedSpansSize(), IsEqual.equalTo(2));
+        }
```

Comment:
I don't know if it's just me, but it looks kinda ungly :P

Reply:
go style :smiley:

Reply:
I was afraid I would hear that :P